apiclient.h

```diff
@@ -0,0 +1,19 @@
+#ifndef APICLIENT_H
+#define APICLIENT_H
+
+#include <memory>
+
+#include "cpprpg.grpc.pb.h"
+#include <grpcpp/grpcpp.h>
+
+class APIClient {
+public:
+  APIClient(std::shared_ptr<grpc::Channel> channel);
+
+  bool useAbility(const int abilityID);
+
+private:
+  std::unique_ptr<PlayerService::Stub> _stub;
+};
```

Comment:
Class needs a comment explaining what it is and how it works (at a high level). Each function should also have a description if it's not immediately obvious from the function name. [This](https://github.com/nullromo/doxygen-example/blob/main/src/dog.hpp) is a decent (if a bit verbose) example.